### PR TITLE
Adds Oppiabot workflow to enable CLA Check in Oppiabot Repository

### PR DESCRIPTION
**Pull request by gp201**
## Explanation

This PR fixes part of #279
Introduces `oppiabot.yml` to run CLA check in [Oppiabot repo](https://github.com/oppia/oppiabot)

## Checklist
- [x] I have successfully deployed my own instance of Oppiabot.
  - You can find instructions for doing this [here](https://github.com/oppia/oppiabot/wiki/Deploying-your-own-instance-of-the-oppiabot).
- [ ] I have manually tested all the changes made in this PR following the [manual tests matrix](https://github.com/oppia/oppiabot/wiki/Manual-Tests-Matrix).

## Note
The secrets have to be added to this repository for the workflow to execute.

### CODE REVIEW

**Comment by gp201** [comment]
@vojtechjelinek @jameesjohn PTAL

File: .github/workflows/oppiabot.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    steps:
+      - uses: actions/checkout@v2
+      - name: CLA Check
+        uses: oppia/oppiabot@1.4.0
+        with:
+          repo-token: ${{secrets.GITHUB_TOKEN}}
+        env:
+          SHEETS_TOKEN: ${{ secrets.SHEETS_TOKEN }}
+          SHEETS_CRED: ${{ secrets.SHEETS_CRED }}
+          SPREADSHEET_ID: ${{ secrets.SPREADSHEET_ID }}
```
Comment (vojtechjelinek) [inline .github/workflows/oppiabot.yml:22]: Add newline at the end of this file.

File: .github/workflows/oppiabot.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+jobs:
+  oppiabot:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: CLA Check
+        uses: oppia/oppiabot@1.4.0
+        with:
+          repo-token: ${{secrets.GITHUB_TOKEN}}
```
Comment (vojtechjelinek) [inline .github/workflows/oppiabot.yml:18]:
```suggestion
          repo-token: ${{ secrets.GITHUB_TOKEN }}
```

File: constants.js [hunk 3]
```diff
@@ -124,6 +124,8 @@ const checksWhitelist = {
   'oppiabot': {
     [openEvent]: [claCheck],
     [reopenEvent]: [],
+    [openEventGithubActions]: [claCheckGithubAction],
+    [reopenEventGithubActions]: [claCheckGithubAction],
```
Comment (vojtechjelinek) [inline constants.js:128]: Above this is also enabled for `reopenEvent`, should we add that too?

**Review by vojtechjelinek — CHANGES_REQUESTED** [review]
Thanks! Left a few comments.

File: .github/workflows/oppiabot.yml (hunk 1 above)
Comment (gp201) [reply .github/workflows/oppiabot.yml]: Done

File: .github/workflows/oppiabot.yml (hunk 2 above)
Comment (gp201) [reply .github/workflows/oppiabot.yml]: Done

File: constants.js (hunk 3 above)
Comment (gp201) [reply constants.js:127]: No, it doesn't have to be added to the `reopenEvent`. I have removed from above.

**Review by vojtechjelinek — APPROVED** [review]
LGTM!

**Review by jameesjohn — APPROVED** [review]
LGTM!

**Pull request merged**